### PR TITLE
Ensure SVGs which provide additional context are accessible

### PR DESCRIPTION
**Pull request by AnthonyLedesma**
WordPress provides the `import { G, Path, SVG }` from `'@wordpress/components` drop-in replacement for the svg element that adds the required accessibility attributes for SVG elements across browsers. 

This pull request converts all SVGs to use the WordPress provided components and removes width and height attributes.

### CODE REVIEW

File: src/blocks/food-and-drinks/food-item/icons.js [hunk 1]
```diff
@@ -10,9 +10,7 @@ const icons = {};
 
 icons.glutenFree = (
 	<SVG
-		height="14"
```
Comment (richtabor) [inline src/blocks/food-and-drinks/food-item/icons.js]: Does removing these height/widths cause deprecation issues?

Comment (richtabor) [reply src/blocks/food-and-drinks/food-item/icons.js]: We should test to confirm; I have a feeling it may.

**Comment by richtabor** [comment]
@paranoia1906 Did you confirm that there were no visual regressions with the icons within the editor UI?

And could you resolve that conflict? Thanks!

File: src/blocks/food-and-drinks/food-item/icons.js (hunk 1 above)
Comment (paranoia1906) [reply src/blocks/food-and-drinks/food-item/icons.js]: Shoot, you are correct. Deprecation's needed all around.

**Comment by paranoia1906** [comment]
@richtabor Blocks have been deprecated and I think I caught all visual regressions.

**Pull request merged**